packages/identity-claims/src/isProfileClaim.js

```diff
@@ -0,0 +1,21 @@
+import { config } from 'meta.js'
+
+/**
+ * Check if a META Identity Claim is a profile claim
+ *
+ * @param  {Object}  claim META Identity Claim object
+ * @return {Boolean}       Profile claim boolean
+ */
+const isProfileClaim = claim => {
+  if (typeof claim === 'undefined' || typeof claim !== 'object') {
+    throw new Error('`claim` is undefined or not of type object.')
+  }
+
+  if (typeof claim.property === 'undefined') {
```

Comment:
Could use `hasOwnProperty` method on the object, as that returns a boolean.
`if(!claim.hasOwnProperty('property'))`

Reply:
Beautiful! Will change 🥇